force-app/main/default/classes/TagAnythingBrowserController.cls

```diff
@@ -0,0 +1,11 @@
+public with sharing class TagAnythingBrowserController {
+
+    @AuraEnabled(Cacheable=true)
+    public static List<Tag_Anything__c> requireTags(String search_key){
+        List<list<SObject>> search_tags = new List<list<SObject>>();
+        search_tags = [FIND :search_key IN ALL FIELDS 
+                       RETURNING Tag_Anything__c(Name, Icon__c, Internal_Value__c, Label_Color__c, Tag_Color__c WHERE Is_Active__c = true ORDER BY Name DESC) 
```

Comment:
@MariaApAb 
`Is_Active__c = true` fix to `Is_Active__c = TRUE` (in caps please)

Question: We want to search the given value (search_key) in ALL fields? That means that if the user gives a color code, they will get a tag with the color (not probable, but could happen). Also the method will found results in Internal_Value__c field. We want that?

Not sure if the SOSL should be updated to search only in `Name` field... What do you think?